packages/enmeshed_runtime_bridge/integration_test/services/facades/consumption/attributes_facade_test.dart

```diff
@@ -1364,10 +1364,10 @@ void run(EnmeshedRuntime runtime) {
       final identityAttribute = identityAttributeResult.value;
 
       final successfulDeletionResult = await sender.consumptionServices.attributes.deleteRepositoryAttribute(attributeId: identityAttribute.id);
-      expect(successfulDeletionResult, isSuccessful<void>());
+      expect(successfulDeletionResult, isSuccessful<VoidResult>());
```

Comment:
```suggestion
      expect(successfulDeletionResult, isSuccessful());
```

I think it should be like that. Same with all other occurences